### ISSUE TITLE
NameError: name 'targetoption' is not defined

### ISSUE THREAD

**Issue by kloggy** [custom-components/places#3]
I have set up two 'places' sensors, one is currently working although the location is 'Home' but the other which is not home is giving this error...

Options used:
```
options: do_not_reorder, place_name, street_number, street, city, postal_code
```

```
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/homeassistant/helpers/entity.py", line 221, in async_update_ha_state
    await self.async_device_update()
  File "/usr/local/lib/python3.6/site-packages/homeassistant/helpers/entity.py", line 349, in async_device_update
    await self.hass.async_add_executor_job(self.update)
  File "/usr/local/lib/python3.6/concurrent/futures/thread.py", line 56, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/local/lib/python3.6/site-packages/homeassistant/util/__init__.py", line 315, in wrapper
    result = method(*args, **kwargs)
  File "/config/custom_components/sensor/places.py", line 432, in update
    self.do_update("Scan Interval")
  File "/config/custom_components/sensor/places.py", line 676, in do_update
    user_display.append(targetoption)
NameError: name 'targetoption' is not defined
```

**Comment by kloggy** [comment]
I *think* this only happens when a restart occurs while the person (or device) is not at home (or probably not in a zone).

**Comment by iantrich** [comment]
fixed in 1.8
